### ISSUE TITLE
Facing keyboard issue on Samsung device

### ISSUE THREAD

**Issue by sainjay** [CaioQuirinoMedeiros/react-native-currency-input#39]
Someone of our users are reporting keyboard disappearing/glitching on the `react-native-currency-input` text inputs.
Check the video below for more context. Any advice to fix this?

https://github.com/user-attachments/assets/717bfe1d-7bb9-41aa-93f6-390762afa3ae

**Comment by CaioQuirinoMedeiros** [comment]
This certainly is not a problem with the library as it just use the TextInput from react-native

**Comment by sainjay** [comment]
Agreed, unable to reproduce it on my end. Would you recommend any solutions for this? @CaioQuirinoMedeiros 
Like `fake-currency-input` maybe?

**Comment by CaioQuirinoMedeiros** [comment]
no, not a clue
